constellations/gemini-admin/src/api.js

```diff
@@ -1,9 +1,44 @@
 "use strict";
 import axios from "axios";
+import moment from "moment";
 
 var BASE_HOST =
     process.env.NODE_ENV == "production" ? "" : "http://localhost:8001";
 
 export default axios.create({
     baseURL: BASE_HOST,
 });
+
+export const executeApiCalls = (apiCalls, successCallback, failCallback) => {
```

Comment:
Could we get more documentation of how to use this utility function?

Something like "Given a list of API calls (promises), execute them in order. If all succeed, execute successCallback. If any fail, execute the failCallback."

^ but maybe more details than that.